obspy/io/xseed/parser.py

```diff
@@ -163,10 +175,20 @@ def read(self, data):
                 download_to_file(url=url, filename_or_buffer=data)
                 data.seek(0, 0)
             elif os.path.isfile(data):
-                # looks like a file - read it
-                with open(data, 'rb') as f:
-                    data = f.read()
-                data = io.BytesIO(data)
+                if is_resp(data):
+                    # RESP filename
+                    with open(data, 'r') as f:
+                        data = f.read()
+                    self._parse_resp(data)
+                    return
+                else:
+                    with open(data, 'rb') as f:
+                        data = f.read()
+                    data = io.BytesIO(data)
```

Comment:
it seems above data is decoded but here it's bytes..? and both ends up in `self._parse_resp(data)`